bucket/filezilla.json

```diff
@@ -15,19 +15,31 @@
     },
     "extract_dir": "FileZilla-3.42.1",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
+        "if (-not (Test-Path \"$persist_dir\\fzdefaults.xml\")) {",
+        "$finalFile = '<FileZilla3><Settings>'",
+        "$finalFile += '<Setting name=\"Config Location\"></Setting>'",
+        "$finalFile += '<Setting name=\"Cache directory\"></Setting>'",
+        "$finalFile += '</Settings></FileZilla3>'",
+        "Set-Content \"$dir\\fzdefaults.xml\" -Value $finalFile -Encoding Ascii",
         "}"
     ],
+    "post_install": [
```

Comment:
Why post install?? 
Just set it ini L20, L21.

Reply:
`pre_install` is for the first install, just create a default config file.
`post_install` is used to keep the location forever.

Reply:
Completely useless.

Do it in pre_install block as suggestions.
Becuase in post_install you edit this file all the time. So when someone do not want his config and cache directory inside install path and change it (in GUI or config file) you will completely break his installation

Reply:
You are right.
But when I changed to `pre_install`, I found that `$dir` is contained version numbers, not `current`.
I can't solve this problem. Please help me.

Reply:
various solutions:

1. Split-Path $dir | Join-Path -ChildPath 'current'
1. $(appdir <manifest> $global)/current'
1. current_dir $dir